src/main/kotlin/uk/gov/justice/hmpps/architecture/model/WMT.kt

```diff
@@ -1,19 +1,44 @@
 package uk.gov.justice.hmpps.architecture.model
 
+import com.structurizr.model.Container
 import com.structurizr.model.Model
 import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
 import com.structurizr.view.ViewSet
 import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.annotations.Tags
 
 class WMT private constructor() {
   companion object : HMPPSSoftwareSystem {
     lateinit var system: SoftwareSystem
+    lateinit var ui: Container
+    lateinit var batchProcessor: Container
+    lateinit var workloadApi: Container
 
     override fun defineModelEntities(model: Model) {
       system = model.addSoftwareSystem(
         "Workload Measurement Tool",
         "(WMT) Helps probation practitioners schedule their time based on service user risk"
       )
+
+      ui = system.addContainer("WMT Web", "View workload data", "Node").apply {
+        url = "https://github.com/ministryofjustice/wmt-web"
+      }
+
+      batchProcessor = system.addContainer("WMT Worker", "Overnight processing of NART report extract", "Node").apply {
+        url = "https://github.com/ministryofjustice/wmt-worker"
+      }
+      workloadApi = system.addContainer("Workload API", "Provides information related to workload data", "Kotlin").apply {
+        url = "https://github.com/ministryofjustice/hmpps-workload"
+      }
+
+      val db = system.addContainer("Database", "Storage for workload data", "PostgreSQL").apply {
+        Tags.DATABASE.addTo(this)
+      }
+
+      ui.uses(db, "connects to")
+      batchProcessor.uses(db, "connects to")
+      workloadApi.uses(db, "connects to")
```

Comment:
Could you add these to the defineRelationships function?

Reply:
Yep